src/regression_tracker.py

```diff
@@ -38,8 +38,10 @@ def _create_regression(self, failed_node, last_successful_node):
         regression = {}
         for field in self._regression_fields:
             regression[field] = failed_node[field]
-        regression['parent'] = failed_node['id']
-        regression['regression_data'] = [last_successful_node, failed_node]
+        regression['data'] = {
+            'testrun_failed': failed_node['id'],
+            'testrun_passed': last_successful_node['id'],
```

Comment:
I think `testrun` is not a very good name, it's not a real word and more importantly it implies this is about tests.  Regressions with node results can be about anything, not just tests.  Something like `node_pass` and `node_fail` might be better.